content/pages/index.md

```diff
@@ -3,7 +3,7 @@ layout: home.html
 body_class: home
 title: Home
 plainlanguage: 11-1-16 Ready for Beth review
-enablewarning: true
+enablewarning: false
```

Comment:
Instead of deleting the banner's markup and style, I figured that the `enablewarning` boolean is there for a reason and that we can save the banner in case something else comes up and we need it again.